mitsurunner.yaml

```diff
@@ -89,6 +96,11 @@ sensor:
     state_topic: $topic_heatexchanger
     filters:    
       - filter_out: nan
+      - exponential_moving_average:
+          # Be careful when tweaking this filter, too small alpha may delay defrosting too much
+          alpha: 0.05
+          send_every: 6 #30s
```

Comment:
These are dependent on time schedules. Could there be calculation based?
Not a blocker now

Reply:
It would be clearer that way, but I think that let's leave it now like this to get the first release done.

Reply:
Agree